vss-core/build.gradle.kts

```
@@ -49,18 +55,22 @@ tasks.withType<Test>().configureEach {
     }
 }
 
-configure<Publish_gradle.PublishPluginExtension> {
+publish {
     mavenPublicationName = "release"
     componentName = "java"
     description = "Vehicle Signal Specification (VSS) Core Module of the KUKSA SDK"
 }
 
 tasks.register("javadocJar", Jar::class) {
     dependsOn("dokkaHtml")
-    from("$buildDir/dokka/html")
+    from("${layout.buildDirectory}/dokka/html")
```

```suggestion
    from("${layout.buildDirectory.get()}/dokka/html")
```
layout.buildDirectory is a Property and needs to be accessed via .get() otherwise it will print "property 'buildDir'"

---

asFile was also missing :) fixed

---

"asFile" should AFAIK not be necessary "layout.buildDirectory.get().toString()" (which is called automatically here) should also already print the correct "dirPath".